apps/clock/style/stopwatch.css

```diff
@@ -14,6 +14,15 @@
 }
 
 /**
+ * font being used in Arabic needs to be
+ * 30% smaller than the usual system font.
+ */
+[lang="ar"] .stopwatch-time {
+  font-size: 6rem;
+  letter-spacing: 0;
+}
```

Comment:
passing by: this seems wrong.
* if the **same** font is used for English and Arabic, then the font should be fixed — not the stylesheets
* if another font is used specifically for the Arabic alphabet, then we must make sure that this font-size adjustment is applied to all languages that use this font — not just Arabic (thinking of Farsi, Urdu…)